treeherder/webapp/api/performance_serializers.py

```diff
@@ -116,8 +117,65 @@ class Meta:
         ]
 
 
+class AlertTaskclusterMetadataSerializer(serializers.ModelSerializer):
+    current_push = serializers.SerializerMethodField()
+    prev_push = serializers.SerializerMethodField()
+
+
+    def get_current_push(self, alert):
+        try:
+            datum_q = PerformanceDatum.objects.filter(
```

Comment:
Wondering why you don't simply query for the job instead of the PerformanceDatum?

Reply:
Good point.

Reply:
Because the performance alerts table doesn't have a direct foreign key to jobs table, it passes through PerformanceDatum first. In other words we don't have data in the alert item to get directly the needed job from Jobs.